google-calendar/1.2/modules/ROOT/pages/google-calendar-connector-reference.adoc

```diff
@@ -71,6 +71,8 @@ Anypoint Connector for Google Calendar (Google Calendar Connector) enables you t
 | Proxy Config a| <<Proxy>> |  Reusable configuration element for outbound connections through a proxy |  |
 | Stream Response a| Boolean |  Whether or not received responses should be streamed |  false |
 | Response Buffer Size a| Number |  The space in bytes for the buffer where the HTTP response will be stored. |  -1 |
+| Access Type a| String |  Indicates whether your application can refresh access tokens when the user is not present at the browser. Valid values are `online` and `offline`. If `offline` is set, a refresh token is retrieved. |  `offline` |
+| Prompt a| String |  Valid values are `consent`, `none`, or `select_account`. Must be set to `consent` to retrieve a refresh token. |  `consent` | 
```

Comment:
The description should first say what the field is or does: "Prompts you to...,"